release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md

```diff
@@ -9,6 +9,57 @@ uid: Cube_Main_Release_10.4.0_other_features_changes
 
 ## Other new features
 
+#### Visual Overview: New BookingData component [ID_33215] [ID_36489]
+
+<!-- MR 10.4.0 - FR 10.3.8 -->
+
+You can now create a special *BookingData* shape and make it display all data associated with a particular booking.
+
+To do so, create a shape with the following shape data fields:
+
+|Shape data field | Value |
+|-------------|---------------|
+| Component   | `BookingData` |
+| Reservation | The ID of the booking<br>Example: `[pagevar:SRMRESERVATIONS_IDOfSelection]` |
+
+A *BookingData* shape will show the following information:
+
+- On the left-hand side, you will find a list of resources used by the booking.
+
+  For every resource, this list shows the following information:
+  
+  - the resource name
+  - an icon indicating the function of the resource
+  - an icon indicating whether the resource is linked to a service definition node
+  - the node label or, if no node label is defined, the name of the function definition
+
+- On the right-hand side, you can select a node or node interface of the resource selected in the list on the left.
```

Comment:
for a left hand side selection, there is only one single node that you can select here. + of course the node interfaces